lib/xcodebuild.js

```diff
@@ -238,9 +238,18 @@ class XcodeBuild {
         'Will build for the default platform instead');
     }
 
-    if (this.realDevice && this.xcodeConfigFile) {
-      this.log.debug(`Using Xcode configuration file: '${this.xcodeConfigFile}'`);
-      args.push('-xcconfig', this.xcodeConfigFile);
+    if (this.realDevice) {
+      if (this.xcodeConfigFile) {
+        this.log.debug(`Using Xcode configuration file: '${this.xcodeConfigFile}'`);
+        args.push('-xcconfig', this.xcodeConfigFile);
+      }
+      if (this.xcodeOrgId && this.xcodeSigningId) {
+        args.push(
+          `DEVELOPMENT_TEAM=${this.xcodeOrgId}`,
+          `CODE_SIGN_IDENTITY=${this.xcodeSigningId}`,
```

Comment:
oh, one thing, probably needs `"`? since it could include spaces

Reply:
exec should escape it automatically as these are separate array items